examples/visualization_demo.ipynb

```diff
@@ -70,21 +70,35 @@
+      "\u001b[1;32m/home/cellefsen/core-ct/examples/visualization_demo.ipynb Cell 7\u001b[0m line \u001b[0;36m4\n\u001b[1;32m      <a href='vscode-notebook-cell://wsl%2Bubuntu/home/cellefsen/core-ct/examples/visualization_demo.ipynb#W6sdnNjb2RlLXJlbW90ZQ%3D%3D?line=0'>1</a>\u001b[0m dir_path \u001b[39m=\u001b[39m os\u001b[39m.\u001b[39mpath\u001b[39m.\u001b[39mjoin(\u001b[39m\"\u001b[39m\u001b[39m.\u001b[39m\u001b[39m\"\u001b[39m, \u001b[39m\"\u001b[39m\u001b[39mexample_core_data\u001b[39m\u001b[39m\"\u001b[39m, \u001b[39m\"\u001b[39m\u001b[39msiegfried_venturelli_core\u001b[39m\u001b[39m\"\u001b[39m)\n\u001b[1;32m      <a href='vscode-notebook-cell://wsl%2Bubuntu/home/cellefsen/core-ct/examples/visualization_demo.ipynb#W6sdnNjb2RlLXJlbW90ZQ%3D%3D?line=1'>2</a>\u001b[0m \u001b[39m# the force option ignores files that produce errors (non-DICOMs)\u001b[39;00m\n\u001b[1;32m      <a href='vscode-notebook-cell://wsl%2Bubuntu/home/cellefsen/core-ct/examples/visualization_demo.ipynb#W6sdnNjb2RlLXJlbW90ZQ%3D%3D?line=2'>3</a>\u001b[0m \u001b[39m# the ignore_file_extensions allows us to read files without the 'dcm' extension\u001b[39;00m\n\u001b[0;32m----> <a href='vscode-notebook-cell://wsl%2Bubuntu/home/cellefsen/core-ct/examples/visualization_demo.ipynb#W6sdnNjb2RlLXJlbW90ZQ%3D%3D?line=3'>4</a>\u001b[0m my_core \u001b[39m=\u001b[39m importers\u001b[39m.\u001b[39;49mdicom(\u001b[39mdir\u001b[39;49m \u001b[39m=\u001b[39;49m dir_path, force \u001b[39m=\u001b[39;49m \u001b[39mTrue\u001b[39;49;00m, ignore_file_extensions \u001b[39m=\u001b[39;49m \u001b[39mTrue\u001b[39;49;00m)\n",
+      "File \u001b[0;32m~/core-ct/src/core_ct/importers.py:85\u001b[0m, in \u001b[0;36mdicom\u001b[0;34m(dir, files, force, ignore_hidden_files, ignore_file_extensions)\u001b[0m\n\u001b[1;32m     83\u001b[0m \u001b[39m# make sure SliceLocation exists in the slice\u001b[39;00m\n\u001b[1;32m     84\u001b[0m \u001b[39mtry\u001b[39;00m:\n\u001b[0;32m---> 85\u001b[0m     \u001b[39mif\u001b[39;00m \u001b[39misinstance\u001b[39m(ds\u001b[39m.\u001b[39;49mSliceLocation, \u001b[39mfloat\u001b[39m):\n\u001b[1;32m     86\u001b[0m         slices\u001b[39m.\u001b[39mappend(ds)\n\u001b[1;32m     87\u001b[0m     \u001b[39melse\u001b[39;00m:\n",
+      "File \u001b[0;32m~/.cache/pypoetry/virtualenvs/core-ct-9AX-Bs9r-py3.10/lib/python3.10/site-packages/pydicom/dataset.py:908\u001b[0m, in \u001b[0;36mDataset.__getattr__\u001b[0;34m(self, name)\u001b[0m\n\u001b[1;32m    906\u001b[0m     \u001b[39mreturn\u001b[39;00m {}\n\u001b[1;32m    907\u001b[0m \u001b[39m# Try the base class attribute getter (fix for issue 332)\u001b[39;00m\n\u001b[0;32m--> 908\u001b[0m \u001b[39mreturn\u001b[39;00m \u001b[39mobject\u001b[39;49m\u001b[39m.\u001b[39;49m\u001b[39m__getattribute__\u001b[39;49m(\u001b[39mself\u001b[39;49m, name)\n",
+      "\u001b[0;31mAttributeError\u001b[0m: 'FileDataset' object has no attribute 'SliceLocation'"
```

Comment:
Right now the notebook displays this `AttributeError: 'FileDataset' object has no attribute 'SliceLocation'` error. Could you rerun the notebook and commit it so it doesn't have this error?

Reply:
Removal of the nrrd file resolved the error